.storybook/CustomTheme.js

```diff
@@ -1,9 +1,8 @@
 import { create } from "@storybook/theming";
-import logo from "./public/nypl-logo.svg";
+import logo from "./public/nypl-reservoir-design-system-logo.svg";
 
 export default create({
   base: "light",
-  brandTitle: "NYPL Design System React Library",
-  brandUrl: "https://nypl.org",
+  brandTitle: "Reservoir Design System React Library",
```

Comment:
We're not adding "NYPL" before Reservoir?

Reply:
Yes, I will add `NYPL` here.